web/changelog.json

```diff
@@ -1,5 +1,15 @@
 [
   {
+    "version": "3.8.x",
```

Comment:
I figure that is best left as the responsibility of whoever cuts the actual release (as well as adding the date).